src/components/Button/Button.jsx

```diff
@@ -71,13 +71,18 @@ const Button = createClass({
 		 * Signature: `({ event, props }) => {}`
 		 */
 		onClick: func,
+		/**
+		 * form element type variations of button. Defaults to 'button' to avoid being triggered by 'Enter' anywhere on the page. Passed through to DOM button.
+		 */
+		type: string,
```

Comment:
It is redundant, but I think this makes it clear that we are defaulting it to `button` aye?